src/patterns/start-pages/index.md.njk

```diff
@@ -0,0 +1,53 @@
+---
+title: Start pages
+description:
+section: Patterns
+theme: Pages
+aliases:
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+If you’re building a digital service, you’ll need to provide a starting point for users on GOV.UK.
+
+{{ example({group: 'patterns', item: 'start-pages', example: 'default', html: true, nunjucks: true, open: true}) }}
```

Comment:
the code for this example is very long, I think we should consider an option to collapse the code on page load, with an option to view it. Probably not this PR though

Reply:
Yeah, I've just realised this is not the format we've followed for the other pages with examples. Will update